docs/whirlpool/docs/03-Whirlpools SDK/03-Position Management/02-Fetch Positions.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: Fetch Positions
+---
+
+# Fetching Positions
+
+Retrieving details about positions held in liquidity pools is an essential part of managing your liquidity and monitoring performance. This guide explains how to use the `fetchPositions()` function to gather information about all active positions held by a given wallet.
```

Comment:
Should we rename this function to something like `fetchPositionsForOwner`?

Reply:
Since @orca-so/whirlpools-client has `fetchPosition(address)` and `fetchAllPositions(addresses) functions

Reply:
I agree that the name `fetchPositionsForOwner()` more accurately describes the functionality of this function.